lib/Elastica/Filter/Bool.php

```diff
@@ -1,6 +1,8 @@
 <?php
 namespace Elastica\Filter;
 
+trigger_error('Use BoolFilter instead. From PHP7 bool is reserved word and this class will be removed in Elastica 4.0', E_USER_DEPRECATED);
```

Comment:
There is already some trigger_errors in another methods. Usually this is not a problem for production, just more log entries.